app/src/main/java/cs5150athletetracking/com/athletetracking/AsyncUploader.java

```diff
@@ -0,0 +1,102 @@
+package cs5150athletetracking.com.athletetracking;
+
+import android.os.AsyncTask;
+import android.util.Log;
+
+import java.io.*;
+import java.net.URL;
+import java.net.HttpURLConnection;
+import java.net.URLEncoder;
+import java.util.Iterator;
+
+import org.json.JSONObject;
+
+
+/**
+ * Created by mag94 on 10/19/15.
+ */
+
+public class AsyncUploader extends AsyncTask<JSONObject, String, Integer> implements ResultCallable {
```

Comment:
I suppose this could work.

So is the convention

AsyncUploader au = new AsyncUploader(JSONObject.... params) {
    @Override
     public void successs(){

    }

    @Override
    public void failure(){
  
    }
}
?